packs/classfeatures/catharsis-emotion.json

```diff
@@ -39,23 +39,23 @@
                 "choices": [
                     {
                         "label": "PF2E.SpecificRule.CatharticMage.Anger.Label",
-                        "value": "anger"
+                        "value": "Anger"
```

Comment:
Ok, I see the problem. If you're going to set the roll option and the localization from this choice then we need an object with both.

```json
"value": {
    "label": "Anger",
    "option": "anger"
}
```
Then reference those with `{item|flags.pf2e.rulesSelections.catharticMageDedication.label}` or `.option`

Reply:
OK, those changes should be in now